imports/ui/Game.jsx

```diff
@@ -152,172 +162,194 @@ class Game extends React.Component {
-      }
-      return <PlayIcon style={{height: 40,
-        width: 40, margin: 10}} color="white" />
-      }
-
-      //======================================================
-      // RENDERING
-      //======================================================
-
-
-      renderPlayerFeatures() {
+      return (
+        <PauseIcon
+          style={{
+            height: 40,
+            width: 40,
+            margin: 10
+          }}
+          color='white'
+        />
+      );
+    }
+    return (
+      <PlayIcon
+        style={{
+          height: 40,
+          width: 40,
+          margin: 10
+        }}
+        color='white'
+      />
+    );
+  }
 
-        const admin = localStorage.getItem('admin');
-        if (admin) {
-          return (
-            <div style={{
+  //======================================================
+  // RENDERING
+  //======================================================
+
+  renderPlayerFeatures() {
+    const admin = localStorage.getItem('admin');
+    if (admin) {
+      return (
+        <div
+          style={{
+            margin: 'auto',
+            display: 'flex'
+          }}
+        >
+          <RaisedButton
+            backgroundColor='#BEDB39'
+            labelColor='white'
+            style={{
               margin: 'auto',
-              display: 'flex'
-            }}>
-
-            <RaisedButton
-              backgroundColor=  "#BEDB39"
-              labelColor="white"
-              style={{
-                margin: 'auto',
-                display: 'flex',
-                width: '90%',
-                height: 60
-              }} label={this.startPauselabel()}
-              onTouchTap={() => this.toggleTimer()} />
-              <RaisedButton
-                backgroundColor=  "#BEDB39"
-                labelColor="white"
-                style={{
-                  margin: 'auto',
-                  display: 'flex',
-                  width: '90%',
-                  height: 60,
-                  color: 'blue'
-                }}
-                label={this.nextGameLabel()}
-                onTouchTap={() => this.nextRound()} />
-
-              </div>
-            )
-          }
-          return null;
-
-        }
-
-        renderTimer () {
-          if (!this.state) return null
-          if (this.props.game.running){
-            return (
-              <div>
+              display: 'flex',
+              width: '90%',
+              height: 60
+            }}
+            label={this.startPauselabel()}
+            onTouchTap={() => this.toggleTimer()}
+          />
+          <RaisedButton
+            backgroundColor='#BEDB39'
+            labelColor='white'
+            style={{
+              margin: 'auto',
+              display: 'flex',
+              width: '90%',
+              height: 60,
+              color: 'blue'
+            }}
+            label={this.nextGameLabel()}
+            onTouchTap={() => this.nextRound()}
```

Comment:
Just FYI, here, you're creating a new function on every `render` call and it's inefficient. You should pass it as `onTouchTap={this.nextRound}` and declaring the function as `nextRound = () => {…}`.